### PR TITLE
[15.0][MIG]barcodes_generator_abstract

### PR DESCRIPTION
**Pull request by manuelregidor**
Superesedes https://github.com/OCA/stock-logistics-barcode/pull/375

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]
LGTM

**Comment by HaraldPanten** [comment]
@pedrobaeza Do you mind merging this one?

File: barcodes_generator_abstract/README.rst [hunk 1]
```diff
@@ -69,20 +69,20 @@ To configure this module, you need to:
 * Select a Nomenclature
 * Create or select a rule
 
-.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png
+.. image:: https://raw.githubusercontent.com/barcodes_generator_abstract/static/description/barcode_rule_tree.png
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/README.rst]: This change is not correct

File: barcodes_generator_abstract/static/description/index.html [hunk 2]
```diff
@@ -428,7 +428,7 @@ <h1><a class="toc-backref" href="#id2">Configuration</a></h1>
 <li>Select a Nomenclature</li>
 <li>Create or select a rule</li>
 </ul>
-<img alt="https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png" src="https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png" />
+<img alt="https://raw.githubusercontent.com/barcodes_generator_abstract/static/description/barcode_rule_tree.png" src="https://raw.githubusercontent.com/barcodes_generator_abstract/static/description/barcode_rule_tree.png" />
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/static/description/index.html]: Incorrect URL change.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
There are incorrect changes in `it.po` in the migration commit. There's also an incorrect added file `Añadimos`.

File: barcodes_generator_abstract/README.rst [hunk 3]
```diff
@@ -69,20 +69,20 @@ To configure this module, you need to:
...
 
-.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png
+.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/15.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png
 
 * For manual generation, set:
     * 'Base set Manually' in 'Generate Type'
     * Set the model
 
-.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_form_manual.png
+.. image:: https://raw.githubusercontent.com/barcodes_generator_abstract/static/description/barcode_rule_form_manual.png
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/README.rst]: It seems you are still editing manually things, and there are a lot of incorrect links. I only point to the first, but you should consider seriously to use the automatic generator or let the bot to do it, but don't do it incorrectly provoking double diff.

File: barcodes_generator_abstract/i18n/it.po [hunk 4]
```diff
@@ -9,15 +9,15 @@ msgstr ""
 "Project-Id-Version: Odoo Server 9.0c\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-11-25 08:40+0000\n"
-"PO-Revision-Date: 2022-05-30 18:05+0000\n"
-"Last-Translator: Samuele Mariani <samuele@ooops404.com>\n"
+"PO-Revision-Date: 2020-11-25 20:36+0000\n"
+"Last-Translator: Francesco Foresti <francesco.foresti@ooops404.com>\n"
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/i18n/it.po]: This file is still being change in the migration commit, which is not correct.

File: barcodes_generator_abstract/readme/CONFIGURE.rst [hunk 5]
```diff
@@ -4,20 +4,20 @@ To configure this module, you need to:
 * Select a Nomenclature
 * Create or select a rule
 
-.. image:: ../static/description/barcode_rule_tree.png
+.. image:: /barcodes_generator_abstract/static/description/barcode_rule_tree.png
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/readme/CONFIGURE.rst]: It was correct as previously. Why are you changing it?

Comment (manuelregidor) [reply barcodes_generator_abstract/readme/CONFIGURE.rst]: It must be pre-commit changing it.

Comment (pedrobaeza) [reply barcodes_generator_abstract/readme/CONFIGURE.rst]: No, pre-commit doesn't touch this.

**Review by pedrobaeza — APPROVED** [review]
OK, let's go with it. The README will be regenerated by the bot.

/ocabot merge nobump

**Comment by manuelregidor** [comment]
@pedrobaeza I've remade both commits so only changes in the right files have been applied. I still don't know why those files got changed.

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@manuelregidor probably you run the readme generator command line `oca-gen-addon-readme` without the proper parameters.

**Pull request merged**